Sources/NIOPosix/PendingDatagramWritesManager.swift

```diff
@@ -693,13 +693,16 @@ final class PendingDatagramWritesManager: PendingWritesManager {
         }
     }
 
-    /// Fail all the outstanding writes. This is useful if for example the `Channel` is closed.
-    func failAll(error: Error, close: Bool) {
-        if close {
-            assert(self.isOpen)
-            self.isOpen = false
-        }
+    /// Close the write manager and fail all the outstanding writes. This is useful if for example the `Channel` is closed.
+    func closeAndFailAll(error: Error) {
+        assert(self.isOpen)
+        self.isOpen = false
 
+        self.failAll(error: error)
+    }
+
+    /// Fail all the outstanding writes. This is useful if for example the `Channel` is closed.
```

Comment:
It might be useful to clean up these doc comments and explain where we might do one and the other.